### PR TITLE
Use UBERON instead of EFO for other organisms in development stage

### PR DESCRIPTION
**Pull request by brianraymor**

### CODE REVIEW

**Comment by pablo-gar** [comment]
Looks good to me. I'll need to do some thinking about how to implement the excluded UBERON terms for development stage.

**Comment by brianraymor** [comment]
@pablo-gar - do you want me to update and move [cellxgene-schema must validate development_stage_ontology_term_id and add human-readable label to development_stage](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-curation/85) from QA to Product Backlog or create a new issue for you?

**Comment by brianraymor** [comment]
@pablo-gar - should we also replace uberon.owl, hsapdv,owl, and mmusdv.owl with composite-metazoan.owl (which includes HsapDv and MmusDv) or wait until a future release?

File: schema/2.0.0/corpora_schema.md [hunk 1]
```diff
@@ -312,7 +312,7 @@ Curators MUST annotate the following columns in the `obs` dataframe:
               <td>A term from the set of <a href="http://www.ontobee.org/search?ontology=MMUSDV&keywords=month-old&submit=Search+terms"> month-old stages</a><br>(e.g. <a href="http://purl.obolibrary.org/obo/MmusDv_0000062">MmusDv:0000062)</a></td>
             </tr>
           </tbody></table>
-          <br> Otherwise, for all other organisms this MUST be the most accurate child of <a href="http://www.ebi.ac.uk/efo/EFO_0000399"<code>EFO:0000399</code></a> for <i>development_stage</i>.
+          <br> Otherwise, for all other organisms this MUST be the most accurate child of <a href="http://purl.obolibrary.org/obo/UBERON:0000105"<code>UBERON:0000105</code></a> for <i>life cycle stage</i>, excluding <a href="http://purl.obolibrary.org/obo/UBERON_0000071"<code>UBERON:0000071</code></a> for <i>death stage</i>, <a href="http://purl.obolibrary.org/obo/UBERON_0035944"<code>UBERON:0035944</code></a> for <i>life-death temporal boundary</i>, and <a href="http://purl.obolibrary.org/obo/UBERON_0035945"<code>UBERON:0035945</code></a> for <i>start of life cycle</i>.
```
Comment (jahilton) [inline schema/2.0.0/corpora_schema.md:315]: http://purl.obolibrary.org/obo/UBERON:0000105 --> http://purl.obolibrary.org/obo/UBERON_0000105
`life-death temporal boundary` & `start of life cycle` are not children of `life cycle stage` so can be removed from the exclusion list (they are children of `life cycle` [UBERON:0000104])

Comment (brianraymor) [reply schema/2.0.0/corpora_schema.md]: Addressed in https://github.com/chanzuckerberg/single-cell-curation/pull/117/commits/dda43abfe2f2f63b2764aa928c5ee348e4c634dc

**Review by jahilton — APPROVED** [review]

**Comment by pablo-gar** [comment]
@brianraymor Yes it'd be helpful to update  the ticket "cellxgene-schema must validate development_stage_ontology_term_id and add human-readable label to development_stage"

>  should we also replace uberon.owl, hsapdv,owl, and mmusdv.owl with composite-metazoan.owl (which includes HsapDv and MmusDv) or wait until a future release?

What do you mean by the composite-metazoan.owl?

**Comment by brianraymor** [comment]
> What do you mean by the composite-metazoan.owl?
See Products http://www.obofoundry.org/ontology/uberon.html and https://github.com/obophenotype/uberon/wiki/Multi-species-composite-ontologies

we could then stop downloading HsapDv.owl and MmusDev.owl and parse those classes from the composite. 

This could also wait until +2.0.0. No rush.

**Comment by pablo-gar** [comment]
Oh got it, probably wait for now then.

**Pull request merged**